src/multi_vector_simulator/utils/constants_json_strings.py

```diff
@@ -134,6 +134,7 @@
 TIMESERIES_PEAK = "timeseries_peak"
 TIMESERIES_TOTAL = "timeseries_total"
 TIMESERIES_AVERAGE = "timeseries_average"
+TIMESERIES_SOC = "timeseries_soc"
```

Comment:
Good thinking - can you also replace the hardcoded strings also in the following occurrences:

![grafik](https://user-images.githubusercontent.com/44204527/103627600-27d88500-4f3e-11eb-9495-400acfc350a5.png)

Reply:
Done!